Kitodo/src/main/java/org/kitodo/services/data/base/SearchService.java

```diff
@@ -290,6 +295,11 @@ public void remove(T baseIndexedBean) throws DataException {
             saveToDatabase(baseIndexedBean);
             removeFromIndex(baseIndexedBean);
             manageDependenciesForIndex(baseIndexedBean);
+            try {
```

Comment:
See my comment in the `save' method.

Reply:
Like above.

Pf course if someone knows better way to handle it I'm open to discuss it :)